slick/src/main/scala/slick/jdbc/H2Profile.scala

```diff
@@ -123,6 +127,32 @@ trait H2Profile extends JdbcProfile {
       override def valueToSQLLiteral(value: UUID) = "'" + value + "'"
       override def hasLiteralForm = true
     }
+    override val instantType : InstantJdbcType = new InstantJdbcType {
+      val formatter = new DateTimeFormatterBuilder()
+                      .append(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss"))
+                      .optionalStart()
+                      .appendFraction(ChronoField.NANO_OF_SECOND, 0, 9, true)
+                      .optionalEnd()
+                      .appendOffset("+HH", "")
```

Comment:
Ditto here. Instant should not be stored only in the UTC zoneoffset, which may also be implied.